docs/acme-divergences.md

```diff
@@ -50,66 +54,73 @@ from the directory instead.
 
 Boulder does not implement orders (previously called `applications`), instead it implements the `new-cert` flow from [draft-ietf-acme-02 Section 6.5](https://tools.ietf.org/html/draft-ietf-acme-acme-02#section-6.5). Instead of authorizations in the order response, Boulder currently uses authorizations that are created using the `new-authz` flow from [draft-ietf-acme-02 Section 6.4](https://tools.ietf.org/html/draft-ietf-acme-acme-02#section-6.4).
 
-## [Section 7.1.4](https://tools.ietf.org/html/draft-ietf-acme-acme-07#section-7.1.4)
+Any identifier of type "dns" in a new-order request MAY have a wildcard domain name as its value, which is not supported by Boulder.
+
+## [Section 7.1.4](https://tools.ietf.org/html/draft-ietf-acme-acme-09#section-7.1.4)
 
 Boulder does not implement the `scope` field in authorization objects.
 
-## [Section 7.2](https://tools.ietf.org/html/draft-ietf-acme-acme-07#section-7.2)
+## [Section 7.2](https://tools.ietf.org/html/draft-ietf-acme-acme-09#section-7.2)
 
 Boulder doesn't implement the `new-nonce` endpoint, instead it responds to `HEAD` requests with a valid `Replay-Nonce` header per [draft-ietf-acme-03 Section 5.4](https://tools.ietf.org/html/draft-ietf-acme-acme-03#section-5.4).
 
-## [Section 7.3](https://tools.ietf.org/html/draft-ietf-acme-acme-07#section-7.3)
+## [Section 7.3](https://tools.ietf.org/html/draft-ietf-acme-acme-09#section-7.3)
 
 Boulder only allows `mailto` URIs in the registrations `contact` list.
 
+Boulder does not handle the stringent "mailto" URI check, in which "Clients MUST NOT provide a "mailto" URL in the "contact" field that contains "hfields" [RFC6068 <https://tools.ietf.org/html/rfc6068>], or more than one "addr-spec" in the "to" component."
+
 Boulder uses an HTTP status code 409 (Conflict) response for an already existing registration instead of 200 (OK). Boulder returns the URI of the already existing registration in a `Location` header field instead of a `Content-Location` header field.
 
 Boulder does not return the `status` field.
 
-Boulder does not implement the `only-return-existing` field.
+Boulder does not implement the `onlyReturnExisting` field.
+
+## [Section 7.3.1](https://tools.ietf.org/html/draft-ietf-acme-acme-09#section-7.3.1)
 
-## [Section 7.3.1](https://tools.ietf.org/html/draft-ietf-acme-acme-07#section-7.3.1)
+Boulder does not implement the `onlyReturnExisting` behaviour and will always create a new account if an account for the given key does not exist.
 
-Boulder does not implement the `only-return-existing` behaviour and will always create a new account if an account for the given key does not exist.
+## [Section 7.3.5](https://tools.ietf.org/html/draft-ietf-acme-acme-09#section-7.3.5)
 
-## [Section 7.3.6](https://tools.ietf.org/html/draft-ietf-acme-acme-07#section-7.3.6)
+Boulder does not provide the value "true" in the "externalAccountRequired" subfield of the "meta" field in the directory object, for new-account requests.  
```

Comment:
Can you also put "true", "externalAccountRequired" and "meta" into backticks:
e.g. 
```
 `"true"`, `"externalAccountRequired"`, and `"meta"`.
```